packages/@react-spectrum/s2/package.json

```diff
@@ -135,9 +135,11 @@
     "@react-aria/i18n": "^3.12.8",
     "@react-aria/interactions": "^3.25.0",
     "@react-aria/live-announcer": "^3.4.2",
+    "@react-aria/overlays": "^3.27.0",
     "@react-aria/utils": "^3.28.2",
     "@react-spectrum/utils": "^3.12.4",
     "@react-stately/layout": "^4.2.2",
+    "@react-stately/menu": "^3.9.3",
```

Comment:
Looks like these dependencies aren't used anywhere?

Reply:
https://github.com/adobe/react-spectrum/pull/7590/files#diff-8cd68236c9fcef8c0976a16402d56ed41af3bbdac40c17b02ff088920c1d51deR52

wanted to revisit this in followup because I'm not sure we should allow menus in a coachmark

Reply:
sorry, wrong one, `@react-aria/overlays` is unused.